scripts/clean

```diff
@@ -43,4 +43,26 @@ if [ ! -z "$1" ]; then
     fi
   done
   rm -f $STAMPS/$1/build_*
+}
+
+if [ "$1" = "--all" ]; then
+  if [ ! -z "$2" ]; then
+    for PROJECT in $(ls -1 projects); do
+      for archfile in projects/$PROJECT/linux/linux.*.conf ; do
+        if [ ! -f "$archfile" ]; then
+          for archfile in projects/$PROJECT/linux/*/linux.*.conf ; do
+            archfile=$archfile
+          done
+        fi
+        echo $archfile
```

Comment:
Is this line debug?

Also what's going on in the second `archfile` for loop? Couldn't you replace line 54 with `break`, rather than assign a value to itself? Also wondering if a single `for` loop would work:
```
for archfile in projects/$PROJECT/linux/linux.*.conf projects/$PROJECT/linux/*/linux.*.conf ; do
```